modules/n1ql/pages/n1ql-language-reference/using-ai.adoc

```diff
@@ -50,19 +46,17 @@ prompt::
 [Required] A natural language request that you want to convert into a SQL++ query.
 
 options::
-[Optional] A JSON object specifying additional <<optional-parameters,options>> to guide the statement execution.
+[Optional] A JSON object specifying additional <<optional-parameters,options>> for the statement.
 +
-By default, the statement uses the `natural_orgid`, `natural_cred`, and `natural_context` parameters.
+By default, the statement uses the xref:n1ql-rest-query:index.adoc#Request[natural_orgid], xref:n1ql-rest-query:index.adoc#Request[natural_cred], and xref:n1ql-rest-query:index.adoc#Request[natural_context] parameters.
```

Comment:
Just reinforcing that these are request-level parameters, they're not part of this statement, which is why they're not described in this page. Also you can link directly to the options!

```suggestion
By default, the statement uses the xref:n1ql-rest-query:index.adoc#natural_orgid[natural_orgid], xref:n1ql-rest-query:index.adoc#natural_cred[natural_cred], and xref:n1ql-rest-query:index.adoc#natural_context[natural_context] request-level parameters.
```